plugins/org.jkiss.dbeaver.net.ssh.ui/src/org/jkiss/dbeaver/ui/net/ssh/SSHUIMessages.properties

```diff
@@ -1,7 +1,7 @@
 
 model_ssh_configurator_button_test_tunnel = Test tunnel configuration
 
-model_ssh_configurator_checkbox_save_pass = Save Password
+model_ssh_configurator_checkbox_save_pass = Save Password/Passphrase
```

Comment:
IMHO it's better to divide this text. As I understand you show password or passphrase, not both of them at once.